app/components/instant-article-field/instant-article-config.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+angular.module('cms.instant_article.config', []).factory('InstantArticleConfig', [function () {
+  var supportedFeatureTypes = [];
+
+  var factory = {};
+
+  factory.setSupportedFeatureTypes = function(featureTypes) {
+    supportedFeatureTypes = featureTypes;
+
+    return this;
+  };
+
+  factory.getSupportedFeatureTypes = function() {
+    return supportedFeatureTypes;
+  };
+
+  return factory;
+}]);
```

Comment:
These are a type of provider under the hood.  They're available at run instead of config. I think the other form is way too verbose for this type, because you're forced to implement $get